docker/install.sh

```diff
@@ -22,7 +22,7 @@ if [[ ! -z "$ROS2_PACKAGES" ]]; then
     source /opt/carma/install/setup.bash
 else
     echo "Sourcing base image for full build..."
-    source /opt/ros/foxy/setup.bash
+    source /opt/ros/humble/setup.bash
```

Comment:
Can we remove the --no-cache option in our docker build script?
https://github.com/usdot-fhwa-stol/autoware.auto/blob/arc-161-ros2-humble/docker/build-image.sh#L131

there are 3 occassions. It is duplicated everywhere and I don't think it really serves us.